config/constants.php

```diff
@@ -1,17 +1,18 @@
 <?php
 
-/*
-  @Author : Mfsi_Annapurnaa
-  @purpose : Constants Declaration
+/**
+ * @Author : Mfsi_Annapurnaa
+ * @purpose : Constants Declaration
  */
 
-// Constant for image path
+//Constant for image path
 define('IMAGEPATH', 'images/profilePicture/', TRUE);
 
-// Constants for db connection	
+//Constants for db connection	
```

Comment:
Now its better to move db configuration setting to a separate db config file instead of putting it in constant file.
db config file can have a different structure may be if you think that would be good.
something like
$db['default']['hostname'] = 'l';
$db['default']['username'] = '';
$db['default']['password'] = '';
$db['default']['database'] = '';

if we define db config setting in array format then in future if required then another array can be set for $db array for replica database to get data from read only database instead of using master database.

// Replica server configuration..
$db['replica']['hostname'] = '';
$db['replica']['username'] = '';
$db['replica']['password'] = '';
$db['replica']['database'] = '';